demos/multi_channel/common/input.cpp

```diff
@@ -491,8 +471,8 @@ bool VideoSourceOCV::isRunning() const {
 template<bool CollectStats>
 void VideoSourceOCV::thread_fn(VideoSourceOCV *vs) {
     while (vs->running) {
-        cv::Mat frame;
-        const bool result = vs->readFrame<CollectStats>(frame);
+        cv::Mat frame = vs->readFrame<CollectStats>();
+        const bool result = frame.data;
```

Comment:
Is this variable even needed?

Reply:
It casts ptr to bool. And since that var is used twice it saves one cast. But I could replace it `frame.data`. I don't see a significant difference.

Reply:
OK, never mind.